e2e/vmcompose/provider.go

```diff
@@ -245,6 +245,20 @@ func (p *Provider) Upgrade(ctx context.Context, cfg types.ServiceConfig) error {
 				}
 			}
 
+			log.Debug(ctx, "Copying prometheus.yaml", "vm", vmName)
+			agentFile := vmAgentFile(instance.IPAddress.String())
```

Comment:
nice, this is perfect, but I took the opportunity to simplify things a bit https://github.com/omni-network/omni/pull/2407/files